addons/website_sale/models/website.py

```diff
@@ -28,7 +28,9 @@ def _default_recovery_mail_template(self):
 
     enabled_portal_reorder_button = fields.Boolean(string="Re-order From Portal")
     salesperson_id = fields.Many2one(
-        string="Salesperson", comodel_name='res.users', domain="[('share', '=', False)]",
+        string="Salesperson",
+        comodel_name='res.users',
+        domain=lambda self: [('share', '=', False)],  # callable to allow reuse in related fields
```

Comment:
I don't think a callable is necessary, a simple list should be enough no?
```suggestion
        domain=[('share', '=', False)],
```

Reply:
Oh yeah, good point, forgot that there was a third option for domains